project/src/test/resources/docker/sql/insert.sql

```diff
@@ -568,7 +645,7 @@ CALL dbo.newEvent(27,
     ARRAY[2],
     ARRAY['Lectures of the SL curricular unit, for the 1718v-1N Class section'],
     ARRAY[2],
-    ARRAY[9],
```

Comment:
Don't forget to revert to `ARRAY[9]` for all of these changes above

Reply:
Was doing this as you commented! :+1: